landingai/predict.py

```diff
@@ -96,28 +95,35 @@ def _check_connectivity(
 
     def _build_default_headers(self, api_key: APIKey) -> Dict[str, str]:
         """Build the HTTP headers for the request to the Cloud inference endpoint(s)."""
-        headers = {
-            "contentType": "application/json",
+        return {
+            "contentType": "multipart/form-data",
```

Comment:
just wondering, why change the contentType to formdata?

Reply:
both work. just think form data is the right content type, since both image file and metadata are form data.